[2025-07-28T18:46:58.898+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T18:46:58.959+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025_07_28_agg [queued]>
[2025-07-28T18:46:58.983+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025_07_28_agg [queued]>
[2025-07-28T18:46:58.984+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-28T18:46:59.457+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-28 18:46:50+00:00
[2025-07-28T18:46:59.512+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=490) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T18:46:59.533+0000] {standard_task_runner.py:63} INFO - Started process 492 to run task
[2025-07-28T18:46:59.509+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'load_data', 'manual__2025_07_28_agg', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpb72d2s7p']
[2025-07-28T18:46:59.556+0000] {standard_task_runner.py:91} INFO - Job 160: Subtask load_data
[2025-07-28T18:46:59.749+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.load_data manual__2025_07_28_agg [running]> on host 7815b21b8845
[2025-07-28T18:46:59.891+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T18:46:50+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025_07_28_agg'
[2025-07-28T18:46:59.893+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T18:47:00.295+0000] {logging_mixin.py:188} INFO - Data loaded into PostgreSQL 'transactions' table.
[2025-07-28T18:47:00.299+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T18:47:00.302+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T18:47:00.329+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=load_data, execution_date=20250728T184650, start_date=20250728T184658, end_date=20250728T184700
[2025-07-28T18:47:00.387+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T18:47:00.472+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-28T18:47:00.501+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
